.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: CI for sbt-riffraff-artifact
+
+on:
+  pull_request:
+  # Manual invocation.
+  workflow_dispatch:
+  push:
+    branches:
+      - $default-branch
+env:
+  FINAL_TEAMCITY_BUILD: 1309
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    # See https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token
+    permissions:
+      # required by guardian/actions-assume-aws-role
+      id-token: write
+      contents: read
+    steps:
+    - name: Checkout branch
+      uses: actions/checkout@v2
+    - uses: guardian/actions-assume-aws-role@v1
+      with:
+        awsRoleToAssume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+    - name: Get node version
+      run: echo NODE_VERSION=$(cat .nvmrc) >> $GITHUB_ENV
+    - uses: actions/setup-node@v2
+      with:
+          node-version: ${{ env.NODE_VERSION }}
```

Comment:
This can be replaced with https://github.com/guardian/actions-setup-node:

```
- uses: guardian/actions-setup-node@v2.4.1
```

`guardian/actions-setup-node` will automatically choose the correct node version by inspecting the `.nvmrc` file and others. It's a fork of `actions/setup-node` (see https://github.com/actions/setup-node/pull/239).